src/lib/style/components/_markdown.scss

```diff
@@ -59,7 +59,7 @@
   word-wrap: break-word;
   overflow-x: auto;
   overflow-y: hidden;
-  line-height: 1.8em;
+  line-height: 1.5em;
```

Comment:
This large line height is what's causing [SWC-6031](https://sagebionetworks.jira.com/browse/SWC-6031). 1.5 is not much different and makes the linked issue much less noticable